imap_processing/mag/l1a/mag_l1a.py

```diff
@@ -40,19 +40,25 @@ def mag_l1a(packet_filepath) -> list[Path]:
         mag_cdf_attrs.mag_l1a_norm_raw_attrs.output(),
         mag_cdf_attrs.mag_l1a_norm_mago_attrs.output(),
         mag_cdf_attrs.mag_l1a_norm_magi_attrs.output(),
+        data_version,
     )
     generated_files += process_and_write_data(
         burst_data,
         mag_cdf_attrs.mag_l1a_burst_raw_attrs.output(),
         mag_cdf_attrs.mag_l1a_burst_mago_attrs.output(),
         mag_cdf_attrs.mag_l1a_burst_magi_attrs.output(),
+        data_version,
     )
 
     return generated_files
 
 
 def process_and_write_data(
-    packet_data: list[MagL0], raw_attrs: dict, mago_attrs: dict, magi_attrs: dict
+    packet_data: list[MagL0],
+    raw_attrs: dict,
+    mago_attrs: dict,
+    magi_attrs: dict,
+    data_version: str,
```

Comment:
It is unclear to me which part of these changes is a temporary. If the addition of `data_version` to this function signature is not temporary, please add it to the docstring.

Reply:
All of it is temporary, including data_version in this change. I need to add a bunch of global attributes so I'm reworking the way this works.